src/app/organization/offices/edit-office/edit-office.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="officeForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Offices</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="officeForm.controls.name.hasError('required')">
+              Office <strong>is required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Parent Office</mat-label>
+            <mat-select required formControlName="parentId">
```

Comment:
add ngif:
this.officeform.hascontrol parentId

Reply:
Changes added. Please have a look